adserver/staff/views.py

```diff
@@ -40,3 +43,70 @@ def get_success_url(self):
                 "advertiser_slug": self.object.slug,
             },
         )
+
+
+class PublisherPayoutView(StaffUserMixin, TemplateView):
+    """
+    A view listing all the payouts that are due in the upcoming month
+
+    Has the following arguments:
+    * all: Show all publishers, not just folks with payouts due
+    * publisher: Filter to a specific publisher
+    * limit: Show only up to ``limit`` publishers, mostly for testing & debugging
+    """
+
+    template_name = "adserver/staff/publisher-payout.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        limit = self.request.GET.get("limit", 50)
```

Comment:
Just a note that this can fail (down below where used) if a non-int is passed. Pretty minor.